docs/FAQ/Accounts.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Accounts & Management
+parent: FAQ
+nav-order: 1
+---
+# Accounts
+
+> Q: Do I need an account?
+>
+> A: An account is not really needed to play on [Zombies](ServernGamemodes.md#Zombies), [PvE](ServernGamemodes.md#PvE), [PvP](ServernGamemodes.md#PvP), [etc](ServernGamemodes.md), **BUT** it is required for [Survival](ServernGamemodes.md#Survival)
+
+## Registering
+
+> Q: How do I register?
+>
+> A: To register an account you **MUST** be in rift(`/rift`), do `/register <password>`, then do `/login <password>`.
+
+> Q: `/register` says I already own an account.
+>
+> A: Aslong as you have logged in **ATLEAST** 1 time into your account, you can simply do `/rename <name the /register command told you>` then `/login` without password(this is of course assuming you have the same IP and Computer), but if you haven't already logged in atleast one time, do `/login <password>` instead of `/login` without a password.
+
+> Q: `/register` says my name is already registered
+>
+> A: Simply do `/rename <new name>` to get another name and register again.
+
+> Q: How do I play with another name but using the same account?
+>
+> A: Join the server with another name then go to `/build` and do `/login <Account Username> <Password>`. `<Account Username>` represents the name you registered with, and `<Password>` represents the password you used when registering.
+>
+> i.e. I registered with a character named AveryClone, and I want to play with the name BlancIsEbic, I'd simply join the server with BlancIsEbic and do `/login AveryClone MyPassword123`.
+
+## Recovery
+
+> Q: I forgot my password
+>
+> A: If you forget your password but you have logged in atleast one time, you can go to `/rift` and then do `/register <password>`, it will tell you that you already have a character named ..., simply do `/rename <name it told you>` then do `/login` with no password.
```

Comment:
```suggestion
> A: If you forget your password but you have logged in at least one time, you can go to `/rift` and then do `/register <password>`, it will tell you that you already have a character named ..., simply do `/rename <name it told you>` then do `/login` with no password.
```